Persistent device id guard (#90)
hpsim/OGL: CMakeLists.txt

  message(FATAL_ERROR "You must source OpenFOAM before building OGL")
endif()

> #Configuration options

Reviewer: ```suggestion
# Configuration options
```
Having the space between # and the comment is preferred programming practice. Space savings are irrelevant here, and why make a comment look like a compiler directive?

Author: Thanks, I accidentally ran `clang-format` on that file, which seems to remove the space here.

add_subdirectory(third_party)

> #If ginkgo wasn't found so far look for the third_party installation

Reviewer: ```suggestion
# If ginkgo is not found at this point, look for the third_party installation
```

    ${CMAKE_CURRENT_BINARY_DIR}/third_party/ginkgo/build/install/lib/cmake)
endif()

> #To interface with ginkgo at least C++ 14 is needed

Reviewer: ```suggestion
# C++ 14 standard or later is required to interface with ginkgo 
```

  add_compile_definitions(linux64)
endif()

#Add a custom command that produces version.cpp, plus a dummy output that's not

> #actually produced, in order to force version.cmake to always be re - run before

Reviewer: ```suggestion
# actually produced, in order to always force re-run version.cmake before
```
